### PR TITLE
update general Mental Health Resources page [#178791388]

### PR DESCRIPTION
**Pull request by bradhenslee**
https://www.pivotaltracker.com/n/projects/2491136/stories/178791388
image

### CODE REVIEW

File: src/data/locales/en/translation.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
       "next": "Next",
       "answer": "This study will only require you to answer weekly survey questions.",
       "dataCollection": "Data Collection, Storage & Privacy",
-      "dataCollectionEncrypted": "Your data will be encrypted and trasferred to a secure cloud server in the U.S.",
+      "dataCollectionEncrypted": "Your data will be encrypted and transferred to a secure cloud server in the U.S.",
```
Comment (bradhenslee) [inline src/data/locales/en/translation.json:114]: unrelated edit, just a typo

**Review by carlosalvarado1988 — APPROVED** [review]

**Review by jwilliam706 — APPROVED** [review]
